examples/rust/src/main.rs

```diff
@@ -1,198 +1,39 @@
-use bitcoin::{util::psbt::serialize::Serialize, Address, Network, PrivateKey};
-use example_common::{build_transaction, get_p2pkh_address, sign_transaction};
-use ic_btc_types::{
-    GetBalanceError, GetBalanceRequest, GetUtxosError, GetUtxosRequest, GetUtxosResponse, OutPoint,
-    SendTransactionRequest, Utxo,
-};
-use ic_cdk::{
-    api::call::RejectionCode,
-    call,
-    export::{
-        candid::{CandidType, Deserialize},
-        Principal,
-    },
-    print, trap,
-};
-use ic_cdk_macros::{init, query, update};
-use std::{cell::RefCell, collections::HashSet, str::FromStr};
+use ic_btc_library::{btc_address_str, get_balance, get_utxos, send, update_transaction};
+use ic_btc_types::Utxo;
+use ic_cdk::trap;
+use ic_cdk_macros::update;
 
-// A private key in WIF (wallet import format). This is only for demonstrational purposes.
-// When the Bitcoin integration is released on mainnet, canisters will have the ability
-// to securely generate ECDSA keys.
-const BTC_PRIVATE_KEY_WIF: &str = "L2C1QgyKqNgfV7BpEPAm6PVn2xW8zpXq6MojSbWdH18nGQF2wGsT";
+const MIN_CONFIRMATIONS: Option<u32> = Some(0);
```

Comment:
This constant is not commented.

Reply:
There wasn't such constant previously so by restoring the previous interfaces, I removed it.